Makefile

```diff
@@ -23,10 +27,7 @@ static:
 	poetry run mypy src
 
 install:
-	poetry install
```

Comment:
That's not the right approach. The default behavior for most users would still be to install canopy itself only.  

The `install-extras` is meant for our CI workflows, which we want to install all the "default" extras on top of base canopy (but not --all-extras`).

Reply:
Changed it